docs/source/configuration/authorization.mdx

```diff
@@ -554,7 +545,7 @@ coprocessor:
       context: true
 ```
 
-then a coprocessor can then receive a request with this format:
+A coprocessor can then receive a request with this format:
```

Comment:
The coprocessor example feels like one that could use `requiresScopes`. Can we reframe it to make it more obvious why you'd use `policy` in some cases and when you _should_ use `requiresScopes` instead? This might be a bigger point we want to make anyway, why does `@requiresScopes` exist if `@policy` is more powerful?